File: Swift/448_find_all_numbers_disappeared_in_an_array.swift
class Solution {
    func findDisappearedNumbers(_ nums: [Int]) -> [Int] {
        let n = nums.count
        let set_nums = Set(nums)
        var answer = Array<Int>()

        for i in 1...n {
            if !set_nums.contains(i) {
                answer.append(i)
            }
        }

        return answer
    }
}

File: Python/448_find_all_numbers_disappeared_in_an_array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        n = len(nums)
        set_nums = set(nums)
        answer = []

        for i in range(1, n + 1):
            if i not in set_nums:
                answer.append(i)

        return answer